src/app/pages/pages-pricing/pages-pricing.component.html

```diff
@@ -162,7 +162,7 @@ <h6 class="text-muted mb-3 mb-sm-4"><a href=""><u>Features to be added to this
                                     </ul>
                                 </div>
                                 <div class="package-actions text-center">
-                                    <a href="/create-account" class="btn btn-lg btn-gradient-primary">
+                                    <a (click)="selectPlan(1)" class="btn btn-lg btn-gradient-primary">
```

Comment:
@Sage-Brezinsky did you make sure that this button still has cursor hand on it.? 
I think it don't.
When you remove href or routerLink from `<a ` then, it don't has hand cursor on it.
Please apply class `cursor-pointer` on it.